compiler/noirc_evaluator/src/ssa/ssa_gen/context.rs

```diff
@@ -265,6 +266,190 @@ impl<'a> FunctionContext<'a> {
         Ok(self.builder.numeric_constant(value, typ))
     }
 
+    /// helper function which add instructions to the block computing the absolute value of the
+    /// given signed integer input. When the input is negative, we return its two complement, and itself when it is positive.
+    fn absolute_value_hlp(&mut self, input: ValueId, sign: ValueId, bit_size: u32) -> ValueId {
```

Comment:
using hlp makes this harder to read -- if its meant to be a helper function, I think its better to just write `helper`

Reply:
Yes it is a helper function, it is only used by the function below.